indico/htdocs/js/indico/modules/registration/form/tpls/fields/dropdown.tpl.html

```diff
@@ -1,23 +1,30 @@
-<ng-form name="nestedForm" ng-init="input = getId(getValue(fieldName))">
-    <select
+<ng-form name="nestedForm" ng-init="input = getId(fieldName)">
+   <select
         id="{{ field.id }}"
         ng-model="input"
         ng-required="field.isRequired"
-        ng-disabled="anyBillableItemPayed(userdata)"
+        ng-disabled="!userdata.manager && selectedItemIsBillable(userdata)"
         ng-class="{hasError: validationStarted && nestedForm.$invalid}"
-        ng-init="selectedItem = getSelectedItem(input) || {}"
-        ng-change="selectedItem = getSelectedItem(input) || {}"
-        ng-click="onSingleFieldItemChange($event)">
+        ng-change="selectedItem = getSelectedItem(input) || {}; onSingleFieldItemChange(selectedItem);">
         <option value="">-- {{ 'Choose a value' | i18n }} --</option>
         <option ng-repeat="item in field.choices | filter:isVisible"
             id="{{ item.id }}"
             value="{{ item.id }}"
-            ng-disabled="!hasPlacesLeft(item, getUserdataValue()) || paymentBlocked(item, userdata)"
-            ng-selected="item.caption == getValue(fieldName)">
+            ng-disabled="!hasPlacesLeft(item, getUserdataValue()) || (!userdata.manager && paymentBlocked(item, userdata))"
+            ng-selected="item.id == getId(fieldName)">
             {{ item.caption }}
             {{ getBillableStr(item, getUserdataValue()) }}
         </option>
     </select>
+    <span ng-if="field.deletedChoice" class="icon-warning deleted-option-warning"
+          data-qtip-style="error"
+          title="{{ 'The currently chosen option is not available anymore. If you unselect it you won\'t be able to choose it back' | i18n }}">
+    </span>
+    <span ng-if="!userdata.manager && userdata.paid && !field.billableDisabled && hasBillableOptions(field)"
```

Comment:
It passing this stuff inside userdata really a good idea? IMHO it'd be cleaner to use separate dicts for the actual form data and metadata like paid, manager, etc